packages/manager/apps/key-management-service/src/data/api/okmsServiceKey.ts

```diff
@@ -51,7 +51,7 @@ export const getOkmsServiceKeyResourceQueryKey = ({
 }: {
   okmsId: string;
   keyId: string;
-}) => [`get/okms/resource/${okmsId}/serviceKey/${keyId}`];
+}) => [`get/okms/resource/${okmsId}/serviceKey/${keyId}?format=JWK`];
```

Comment:
we should have 2 different queries between `display a key` and `download a key`, as we don't need the `jwk` data on the dashboard.